app/views/groups/_info_window.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2><%= group.name %></h2>
```

Comment:
use html.slim template instead of html.erb

Reply:
I've never used that before so I might make some syntax errors out the gate. It looks nice though. I'll go over the docs a bit today and make the changes.